src/main/java/codeu/controller/ActivityFeedServlet.java

```diff
@@ -64,22 +77,24 @@ public void doGet(HttpServletRequest request, HttpServletResponse response)
     List<Conversation> conversations = conversationStore.getAllConversations();
     request.setAttribute("conversations", conversations);
 
+    // Get and set Messages
+    List<Message> messages = messageStore.getAllMessages();
+    request.setAttribute("messages", messages);
+
     // Make, sort, and set the event map
-    Map<Instant, Object> unsortedEventsMap = new HashMap<Instant, Object>();
```

Comment:
Unrelated to this PR: you don't really need the extra `Instant` field of each record, you can just have the comparator check the object's `getCreationTime` method since they all implement it.

Reply:
Would this mean that List<Object> would be better than the map?

Reply:
Well you still need sorting. If you don't want to use a list and sort it, you could consider a different data structure with inherent sorting, maybe a `HashSet`?

Reply:
Thanks, I'll look into it. Also, once the Event class is implemented would it be better to keep using a List then just traverse it in reverse? That way there's no need for sorting.

Reply:
You'll still need to keep the list sorted on the event time.
Another alternative is to make sure things are inserted into the data store in order (which I think they are), so that you can just get the lists of convos, chats, and users, then do an efficient time-based merge, which is just the merge step of a [merge sort](https://en.wikipedia.org/wiki/Merge_sort).

Reply:
I hadn't considered merging before! I think for now I'll traverse in reverse since they're stored in order and then figure out how to merge, thanks.